modules/tensor_mechanics/include/materials/SingleVariableReturnMappingSolution.h

```diff
@@ -105,6 +105,24 @@ class SingleVariableReturnMappingSolution
                                    const Real residual,
                                    const Real reference_residual);
 
+  /**
+   * Output information for a single iteration step to build the convergence history of the model.
+   * This method is duplicative of outputIterationStep, but matches the signature of
+   * ADSingleVariableReturnMappingSolution to allow for consistant code impelementation.
+   * @param iter_output            Output stream
+   * @param effective_trial_stress Effective trial stress
+   * @param residual               Current value of the residual
+   * @param reference              Current value of the reference quantity
+   */
+  virtual void outputIterationStep(std::stringstream * iter_output,
+                                   const Real & effective_trial_stress,
+                                   const Real & scalar,
+                                   const Real reference_residual)
+  {
+    return outputIterationStep(
```

Comment:
There are other inconsistencies that'll need to be rectified how you'd like @dschwen between `ADSingleVariableReturnMappingSolution` and `SingleVariableReturnMappingSolution`, mostly with pass by references (or not), but this will allow me to implement what I need now for this method.

Reply:
Ok, the way to go is to have consistent pass by const reference. This will be optimized by the compiler for built in datatypes like `double` and will be way more efficient for `ADReal`

Reply:
Yup, makes sense to me, but it may break some internals.